packages/core-flows/src/payment/workflows/refund-payment.ts

```diff
@@ -0,0 +1,17 @@
+import { WorkflowData, createWorkflow } from "@medusajs/workflows-sdk"
+import { refundPaymentStep } from "../steps/refund-payment"
+
+export const refundPaymentWorkflowId = "refund-payment-workflow"
+export const refundPaymentWorkflow = createWorkflow(
+  refundPaymentWorkflowId,
+  (
+    input: WorkflowData<{
+      payment_id: string
+      created_by?: string
+      amount?: number
+    }>
+  ) => {
+    const payment = refundPaymentStep(input)
```

Comment:
nit: return refundPaymentStep(input)

Reply:
question: How does this work in case someone appends a step to `refundPaymentStep`? This is why I chose to separate the two.